scripts/collect_docker_version_files.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+  
+DOCKER_IMAGE=$1
+TARGET_PATH=$2
+
+[ -z "$TARGET_PATH" ] && TARGET_PATH=./target
+
+DOCKER_IMAGE_NAME=$(echo $DOCKER_IMAGE | cut -d: -f1)
+DOCKER_CONTAINER=$DOCKER_IMAGE_NAME
+TARGET_VERSIONS_PATH=$TARGET_PATH/versions/dockers/$DOCKER_IMAGE_NAME
+
+[ -d $TARGET_VERSIONS_PATH ] && rm -rf $TARGET_VERSIONS_PATH
+mkdir -p $TARGET_VERSIONS_PATH
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
+docker create --name $DOCKER_CONTAINER --entrypoint /bin/bash $DOCKER_IMAGE > /dev/null 2>&1
```

Comment:
>$DOCKER_CONTAINER [](start = 21, length = 17)

If this container exists before this command, will it fail?
If yes, clean it before this command.

Reply:
Yes, it will be failed. Will fix it, thanks.

---
In reply to: [534558221](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r534558221) [](ancestors = 534558221)

Reply:
I don't know what is behavior if you redirect the error (ignore it).
Better explicitly clean it before this command.

Ignoring error is dangerous. If we know some failure case, just fix the specific case.

---
In reply to: [534576919](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r534576919) [](ancestors = 534576919,534558221)

Reply:
I will change to print the message and error.

---
In reply to: [537935627](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r537935627) [](ancestors = 537935627,534576919,534558221)